flocker/control/_protocol.py

```diff
@@ -81,7 +88,8 @@ class ClusterStatusCommand(Command):
     in the convergence agent during startup.
     """
     arguments = [('configuration', DeploymentArgument()),
-                 ('state', DeploymentArgument())]
+                 ('state', DeploymentArgument()),
+                 ('eliot_context', Unicode())]
```

Comment:
I kinda feel the whole "let's patch a `Logger`" idiom was a mistake and that really the default in tests should just be "capture all log messages generated by the test".